exercises/hello-world/makefile

```diff
@@ -4,10 +4,17 @@ CFLAGS += -Wextra
 CFLAGS += -pedantic
 CFLAGS += -Werror
 
+VFLAGS  = --quiet
+VFLAGS += --tool=memcheck
+VFLAGS += --leak-check=full
+VFLAGS += --error-exitcode=1
```

Comment:
This seems like something only needed for a script, does not help the user.

Reply:
Thought this might be useful for potential future change to `run-tests` to change from explicitly calling `valgrind` to instead using this `make` target. This would allow flexibility in the case that an exercise requires special consideration WRT `valgrind`. Is there any negative to using it?

Reply:
With the added two `echo` statements this now also means that the second doesn't get output in case of failure.

Reply:
LGTM, if the second only gets printed on a success then you could say `Memory check passed.`
I'm fine with the other flags, just wanted the discussion.  No strong feeling.